style.css

```diff
@@ -0,0 +1,64 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
```

Comment:
Well done here as well , but you still can improve he styling add box-shadow to your buttons , use the same font and margins ,always do your best to replicate the given design

![buttons box shadow](https://user-images.githubusercontent.com/59707859/226885306-3e326dfb-8d55-4d5c-8d1f-ed9a6247d1d4.PNG)

![book stor student](https://user-images.githubusercontent.com/59707859/226885351-266c8b26-26a2-4834-a182-0442a773890c.PNG)